src/coreclr/tools/Common/TypeSystem/Interop/IL/MarshalHelpers.cs

```diff
@@ -452,14 +455,6 @@ public static partial class MarshalHelpers
             }
             else if (type.IsSzArray)
             {
-#if READYTORUN
```

Comment:
Is it okay to delete these before #69758 is done?

Reply:
Yes, it is okay to delete this. We are only generating no-op blittable marshallers for R2R now. Arrays are never blittable.